udata/core/dataset/croquemort.py

```diff
@@ -19,6 +19,15 @@
 TIMEOUT_LOG_MSG = 'Timeout connectin to Croquemort'
 
 
+def has_checked(response):
+    if response.status_code == 404:
```

Comment:
I just realized that maybe we should use requests codes:

```python
if response.status_code == requests.status_codes.codes.not_found:
```

Same for 503 => `codes.service_unavailable`